electrum/network.py

```diff
@@ -684,7 +695,14 @@ async def set_parameters(self, net_params: NetworkParameters):
                 int(proxy['port'])
         except Exception:
             return
-        self.config.NETWORK_AUTO_CONNECT = net_params.auto_connect
+        auto_connect = net_params.auto_connect
+        if auto_connect and net_params.oneserver:
+            # enabling both oneserver and auto_connect doesn't really make sense
+            # assume oneserver is enabled for privacy reasons, disable auto_connect
+            self.logger.warning(f'both "oneserver" and "auto_connect" options enabled, disabling "auto_connect".')
+            auto_connect = False
```

Comment:
I don't know... I think respecting the auto_connect setting and disabling oneserver might be better.
If they did not disable auto_connect, what's the guarantee that the current server is selected correctly? If it is effectively a "random" server, now disabling auto_connect makes sure we won't switch away from it.

If `auto_connect and oneserver` we should either transition to `auto_connect and not oneserver`
or just kill the network by transitioning to `server==localhost:1:s and not auto_connect and not oneserver`.

Reply:
>  or just kill the network by transitioning to server==localhost:1:s and not auto_connect and not oneserver.

well actually maybe to `server==localhost:1:s and not auto_connect and oneserver`